# List of waypoints forming a navigation path
float64[] latitudes     # Array of waypoint latitudes
float64[] longitudes    # Array of waypoint longitudes
uint32 total_waypoints  # Total number of waypoints in the path
uint32 current_index    # Index of the current target waypoint
